lib/services/upstart.h

```diff
@@ -23,7 +23,7 @@
 #  include "crm/services.h"
 
 G_GNUC_INTERNAL GList *upstart_job_listall(void);
-G_GNUC_INTERNAL int upstart_job_exec(svc_action_t * op, gboolean synchronous);
+G_GNUC_INTERNAL gboolean upstart_job_exec(svc_action_t * op);
```

Comment:
Ah, you can see how I like this cut-several-changes-back-to-commits process :-)
I've missed this.